tests/AssertTest.php

```diff
@@ -633,7 +643,7 @@ public function testNullOr($method, $args, $success, $multibyte = false, $minVer
      */
     public function testNullOrAcceptsNull($method)
     {
-        call_user_func(array('Webmozart\Assert\Assert', 'nullOr'.ucfirst($method)), null);
+        call_user_func(array('Webmozart\Assert\Assert', 'nullOr'.ucfirst($method)), null, null, null);
```

Comment:
Could you please tell me why this is needed?

Reply:
This is because `Assert::nullOrBetween()` requires, for instance, 3 parameters.

This was not caught before because `__callStatic` is reeeeally lax.